### ISSUE TITLE
CDN: `Failed to find a valid digest in the 'integrity' attribute for resource`

### ISSUE THREAD

**Issue by garrytrinder** [OfficeDev/microsoft-teams-library-js#1404]
I copied the script tag from the homepage, but it does not load.

```html
<!-- Microsoft Teams JavaScript API (via CDN) -->
<script
  src="https://res.cdn.office.net/teams-js/2.4.1/js/MicrosoftTeams.min.js"
  integrity="sha384-l5BX+R3H+6sfSa7KKsqzBsWKsJ9JE9O0a1DqQKtM/RbE1Gb6N6bGYlNJt4EIZZ6X"
  crossorigin="anonymous"
></script>
```

<img width="702" alt="image" src="https://user-images.githubusercontent.com/11563347/196404987-ac8306dd-f906-4ca1-ad1e-500303bcd1c4.png">

**Comment by AE-MS** [comment]
Hi @garrytrinder, thank you so much for reporting this issue.

It looks like the hash value provided on [the npm page](https://www.npmjs.com/package/@microsoft/teams-js) for the package is out of date and does not match what is in the [package's readme file](https://github.com/OfficeDev/microsoft-teams-library-js/tree/main/packages/teams-js). The latter is correct.

I will see about getting the npm page's updated to match. In the meantime, can you try the value provided in the package readme file and see if that fixes it for you?

```
<!-- Microsoft Teams JavaScript API (via CDN) -->
<script
  src="https://res.cdn.office.net/teams-js/2.4.1/js/MicrosoftTeams.min.js"
  integrity="sha384-f54aC8b3HUFudeEk+QUkv5RpZUpPr2g7TdP4+jnDAMoJSmt+PbMdUV+zGTYCQY1c"
  crossorigin="anonymous"
></script>
```

**Comment by jadahiya-MSFT** [comment]
Hi @garrytrinder this issue has been fixed with version 2.4.2. We've updated the README with the correct hash value on both the npm page and on the Github page. Let me know if you run into any other problems.

**Comment by jadahiya-MSFT** [comment]
Hi @garrytrinder this issue has been fixed with version 2.4.2. We've updated the README with the correct hash value on both the npm page and on the Github page. Let me know if you run into any other problems.

**Comment by garrytrinder** [comment]
Great, thanks for the update 👍
